backend/src/main/routes/AccountRoute.ts

```diff
@@ -188,6 +205,7 @@ export default class AccountRoute extends CommonRoutesConfig {
       });
     this.getApp()
       .route(`/accounts/client/:email`)
+      .all(checkJwt, checkRole(new Set([Roles.BUSINESS])))
```

Comment:
similar issue to the comment mentioned above

Reply:
Fixed. Added verification for it